server/src/main/scala/drt/staff/ImportStaff.scala

```diff
@@ -0,0 +1,28 @@
+package drt.staff
+
+import org.joda.time.{DateTime, DateTimeZone}
+import services.SDate
+import services.SDate.implicits._
+
+import scala.util.parsing.json.JSON
+
+object ImportStaff
+{
+  def staffJsonToShifts(staffJson: String): String = {
+    val parsed: Seq[String] = JSON.parseFull(staffJson) match {
+      case Some(s: List[Map[String, Any]]) =>
+        s.zipWithIndex.map{
+          case (shift, index) =>
+            //The client deals in local time, and these shifts are sent to the client as strings with no timezone for now.
+            //TODO: store shifts not as strings.
+            val shiftStartDate = new DateTime(shift("shift_start").toString).withZone(DateTimeZone.forID("Europe/London"))
+            val shiftsEndDate = shiftStartDate.addMinutes(15)
+            f"shift$index, ${shift("terminal")}, ${shiftStartDate.ddMMyyString}, ${shiftStartDate.getHours()}%02d:${shiftStartDate.getMinutes()}%02d, ${shiftsEndDate.getHours()}%02d:${shiftsEndDate.getMinutes()}%02d, ${shift("staff")}"
+        }
+      case error =>
+        println(s"got some bollocks: $error, $staffJson")
```

Comment:
if any keys are missing on any part of it, then I think this'll fail with an exception. unlikely, as we control the poster. but worth thinking about.